pkg/sql/logictest/testdata/logic_test/database

```diff
@@ -361,3 +366,24 @@ database_name       owner     primary_region  secondary_region  regions  surviva
 ifnotexistsownerdb  testuser  NULL            NULL              {}       NULL
 
 subtest end
+
+subtest regression_105906
+
+statement ok
+CREATE SCHEMA regression_105906
```

Comment:
Can you also add test cases where database name need to be double quoted? for example "regression 105906" ? Thanks

Reply:
Added!